functions/src/emails.ts

```diff
@@ -0,0 +1,85 @@
+import { sendgrid } from "./apis";
+import Lob from "./lob";
+
+export function notifyUserOnLetterCreate({
+  to,
+  lobResponses,
+}: {
+  to: string;
+  lobResponses: Lob.CreateResponse[];
+}): Promise<void> {
+  const responseSummaries = lobResponses
+    .map((lobResponse: any) => {
+      return `Sent to: ${lobResponse.to.name}
+Expected Delivery: ${lobResponse.expected_delivery_date}
+Preview: ${lobResponse.url}`;
+    })
+    .join("\n\n");
+
+  const text = `You sent some letters!\n\n${responseSummaries}`;
+
+  const htmlResponseSummaries = lobResponses
+    .map((lobResponse: any) => {
+      return `<li> ${
+        lobResponse.to.name
+      } - expected delivery: ${lobResponse.expected_delivery_date.substring(0, 10)} <a href="${
+        lobResponse.url
+      }">(preview)</a></li>`;
+    })
+    .join("\n");
+
+  const html = `Your letters have been sent to the printer!\n\n
+  
+  <ul>${htmlResponseSummaries}</ul>
+  You should get a follow-up email in a few days when the letters are close to being delivered
+  `;
+
+  const msg = {
+    to,
+    from: "defund12@blackmad.com",
```

Comment:
Can we put this address in the config? I also see that it's duplicated below

Reply:
moved